scrapy/pipelines/__init__.py

```diff
@@ -22,3 +22,21 @@ def _add_middleware(self, pipe):
 
     def process_item(self, item, spider):
         return self._process_chain('process_item', item, spider)
+
+
+class ItemPipeline(object):
+    """Base class for scrapy item pipelines.
```

Comment:
Yes I thought of that as well. But I tried to keep the implementation as close to the way the `Spider` base class is implemented. But now I see that there are more `Base*` classes in the scrapy code, so I'll change it to `BaseItemPipeline`.